### ISSUE TITLE
unit test KO (ScenarioInitiatorUT)

### ISSUE THREAD

**Issue by sgrillon14** [NoraUi/NoraUi#43]
[ERROR] There are test failures.
Please refer to /home/travis/build/NoraUi/NoraUi/target/surefire-reports for the individual test results.
Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
Command was /bin/sh -c cd /home/travis/build/NoraUi/NoraUi && /usr/lib/jvm/java-8-oracle/jre/bin/java -Dfile.encoding=UTF-8 -javaagent:/home/travis/.m2/repository/org/jacoco/org.jacoco.agent/0.7.9/org.jacoco.agent-0.7.9-runtime.jar=destfile=/home/travis/build/NoraUi/NoraUi/target/test-results/coverage/jacoco/jacoco.exec -jar /home/travis/build/NoraUi/NoraUi/target/surefire/surefirebooter3942140096945827967.jar /home/travis/build/NoraUi/NoraUi/target/surefire 2018-03-29T21-39-05_475-jvmRun1 surefire143197456744254289tmp surefire_02617730578104669031tmp
Error occurred in starting fork, check output in log
Process Exit Code: 255
Crashed tests:
com.github.noraui.main.ScenarioInitiatorUT
org.apache.maven.surefire.booter.SurefireBooterForkException: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
Command was /bin/sh -c cd /home/travis/build/NoraUi/NoraUi && /usr/lib/jvm/java-8-oracle/jre/bin/java -Dfile.encoding=UTF-8 -javaagent:/home/travis/.m2/repository/org/jacoco/org.jacoco.agent/0.7.9/org.jacoco.agent-0.7.9-runtime.jar=destfile=/home/travis/build/NoraUi/NoraUi/target/test-results/coverage/jacoco/jacoco.exec -jar /home/travis/build/NoraUi/NoraUi/target/surefire/surefirebooter3942140096945827967.jar /home/travis/build/NoraUi/NoraUi/target/surefire 2018-03-29T21-39-05_475-jvmRun1 surefire143197456744254289tmp surefire_02617730578104669031tmp
Error occurred in starting fork, check output in log
Process Exit Code: 255
Crashed tests:
com.github.noraui.main.ScenarioInitiatorUT
	at org.apache.maven.plugin.surefire.booterclient.ForkStarter.fork(ForkStarter.java:686)
	at org.apache.maven.plugin.surefire.booterclient.ForkStarter.fork(ForkStarter.java:535)
	at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:280)
	at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:245)
	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeProvider(AbstractSurefireMojo.java:1124)
	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:954)
	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute(AbstractSurefireMojo.java:832)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:309)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:194)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:107)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:955)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:290)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:194)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)

**Comment by sgrillon14** [comment]
Test OK but logs is not print. problem from this:

```
    @After
    public void restoreStreams() {
        System.setOut(System.out);
        System.setErr(System.err);
    }
```

**Comment by sgrillon14** [comment]
fix with this commit [5ff9607](https://github.com/NoraUi/NoraUi/commit/5ff960734d050425697d097c8d71d8c383c7d920)
